pkgs/development/python-modules/aioprometheus/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, orjson
+, quantile-python
+, aiohttp
+, starlette
+, quart
+, pytestCheckHook
+, asynctest
+, httpx
+, fastapi
+, uvicorn
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aioprometheus";
+  version = "22.5.0";
+  format = "setuptools";
+
+  disable = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "claws";
+    repo = "aioprometheus";
+    rev = version;
+    hash = "sha256-oXLKJDCJFRcR+yOXpTVuQgrnOfPRnS8g51Aci/qMOOw=";
+  };
+
+  propagatedBuildInputs = [
+    orjson
+    quantile-python
+    aiohttp
+    starlette
+    quart
```

Comment:
Some of those [requirements](https://github.com/claws/aioprometheus/blob/22.5.0/setup.py#L56-L60) should be  handled with `passthru.optional-dependencies`.

Reply:
Fixed